src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -743,6 +752,54 @@ namespace AppInstaller::CLI::Workflow
         context.Add<Execution::Data::InstalledPackageVersion>(context.Get<Execution::Data::Package>()->GetInstalledVersion());
     }
 
+    void ReadAdditionalData(Execution::Context& context)

```

Comment:
Overall this pattern doesn't seem like what we talked about.

Reply:
Updated the changes per our discussion..